packages/airnode-validator/src/ois/ois.ts

```diff
@@ -154,12 +154,27 @@ const ensurePathParametersExist: ValidatorRefinement<SchemaType<typeof pathsSche
 
 export const pathsSchema = z.record(pathSchema).superRefine(ensurePathParametersExist);
 
-export const apiSpecificationSchema = z.object({
-  components: apiComponentsSchema,
-  paths: pathsSchema,
-  servers: z.array(serverSchema),
-  security: z.record(z.tuple([])),
-});
+export const apiSpecificationSchema = z
+  .object({
+    components: apiComponentsSchema,
+    paths: pathsSchema,
+    servers: z.array(serverSchema),
+    security: z.record(z.tuple([])),
+  })
+  .superRefine((apiSpecification, ctx) => {
```

Comment:
I know the issue doesn't mention this, but can we also check it the other way around? (that every scheme from `components.securitySchemes` is enabled in the `security`)?